src/App.css

```diff
@@ -1,3 +1,269 @@
 .App {
+  background-color: #fafafa;
+  font-family: Montserrat, sans-serif;
+}
+
```

Comment:
- [ ] Your "RobotoSlab" font won't render because it does not exist.  The correct name is "Roboto Slab".  Please, kindly correct all the occurrences on your CSS, so the "Roboto Slab" font renders correctly:

![image](https://user-images.githubusercontent.com/57421823/166853006-112bef78-a425-4345-b75b-c9be0f0dc3d9.png)

![image](https://user-images.githubusercontent.com/57421823/166853068-61e221a0-ea96-47c3-a475-b88f39a0623b.png)